index.bs

```diff
@@ -75,7 +75,7 @@ type: dfn
   transport or it isn't -- and encourage developers to securely load any mixed content that is
   necessary for their webpage to function properly.
 
-  Mixed Content Level 2 therefore updates and extends [[!mixed-content]] to provide users with
+  So this specification was updated to provide users with
   better security and privacy guarantees and a better security UX, while minimizing
   breakage. Instead of advising browsers to simply strictly block all mixed content, the core idea
   of Level 2 is <i>mixed content autoupgrading</i>. That is, mixed content that user agents are not
```

Comment:
Maybe switch this to "the core idea of the updated specification is"

Reply:
> For the most part looks good to me, left a comment re: a leftover usage of "Level 2"

Thanks for catching that. Please review bad1d13 for some changed language around that.

Reply:
Thanks! The last commit looks good to me.